CHANGELOG.md

```diff
@@ -66,7 +82,7 @@
 * ANR or NDK detection warnings can be suppressed (using `enabledErrorTypes`) when plugin is excluded.
   [#1832](https://github.com/bugsnag/bugsnag-android/pull/1832)
 
-* Support using regexes to define redacted keys and discard classes.
+* Support using regexes to define redacted keys and discardClasses classes.
```

Comment:
This looks like a find-and-replace glitched. But also I'm not quite sure why this is listed in 5.30.0?

Reply:
This looks a lot like a merge-mistake. I'll remove it.